sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -107,15 +112,12 @@ class FrameState(object):
 
   def __init__(self, f, local_vars=None, stack=()):
     self.f = f
-    if sys.version_info[0] >= 3:
-      self.co = f.__code__
-    else:
-      self.co = f.func_code
+    self.co = f.__code__
```

Comment:
<!--new_thread; commit:b015963f671ceb39f8033014eb6d4e46a4a734bc; resolved:0-->
How can we drop the python 2 branch? Does `self.co = f.__code__` work for all versions?

Reply:
This syntax works for Python 2.6+ so it shouldn't make a difference